Update schema descriptions, examples and regenerate the docs (#74)
EnterpriseDB/terraform-provider-biganimal: docs/resources/region.md

# biganimal_region (Resource)

> The region resource is used to manage regions for a given cloud provider. See https://www.enterprisedb.com/docs/biganimal/latest/getting_started/activating_regions/ for more details.

Reviewer: ```suggestion
The region resource is used to manage regions for a given cloud provider. See [Activating regions }(https://www.enterprisedb.com/docs/biganimal/latest/getting_started/activating_regions/) for more details.
```

Author: Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660

!> Creation of region resources is not supported for now.

## Example Usage
```terraform
terraform {
  required_providers {
    biganimal = {
      source  = "biganimal"
      version = "0.3.1"
    }
  }
}

resource "biganimal_region" "this" {
  cloud_provider = "aws"
  region_id      = "eu-west-1"
}

output "region_status" {
  value = resource.biganimal_region.this.status
}

output "region_name" {
  value = resource.biganimal_region.this.name
}

output "region_continent" {
  value = resource.biganimal_region.this.continent
}
```

<!-- schema generated by tfplugindocs -->
## Schema

### Required

> - `cloud_provider` (String) Cloud Provider. e.g. "aws", "azure".

Reviewer: ```suggestion
- `cloud_provider` (String) Cloud Provider. For example, "aws" or "azure".
```

Author: e.g. -> For example, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a
Adding `or`, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/9b68f4682dd7b93a097f749e6d7aecac4dfa74b1

> - `region_id` (String) Region ID of the region. e.g. "germanywestcentral" in the azure cloud provider, "eu-west-1" in the aws cloud provider.

Reviewer: ```suggestion
- `region_id` (String) Region ID of the region. For example, "germanywestcentral" in the Azure cloud provider or "eu-west-1" in the AWS cloud provider.
```

Author: Spelling correction of the cloud providers, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/fd43c8ae86d3aad7abff08c0cece786c2e54a182
Adding `or`, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/460be1df55a563f5ac981ad7e7a2d33021b8e941

### Optional

> - `status` (String) Region Status of the region. e.g. "ACTIVE", "INACTIVE" or "SUSPENDED".

Reviewer: ```suggestion
- `status` (String) Region Status of the region. For example, "ACTIVE", "INACTIVE", or "SUSPENDED".
```

Author: `e.g.` -> `For example,` fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a
Serial comma, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/a7e9930ede32c6adc1fc0810c56a60d8391f3fb3

- `timeouts` (Block, Optional) (see [below for nested schema](#nestedblock--timeouts))

### Read-Only

> - `continent` (String) Continent that region belongs to. e.g. "Asia","Australia","Europe".

Author: e.g. -> For example, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a
Serial comma and adding the whitespaces, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/a7e9930ede32c6adc1fc0810c56a60d8391f3fb3

- `id` (String) The ID of this resource.

> - `name` (String) Region Name of the region. e.g. "Germany West Central", "EU West 1".

Reviewer: ```suggestion
- `name` (String) Region Name of the region. For example, "Germany West Central" or "EU West 1".
```

Author: e.g. -> For example, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a
`,` -> `or`, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/02511f70c33c45857d2e1be15e48ded86b5c6403
